java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CommonRequestHandler.java

```diff
@@ -227,15 +227,19 @@ public HttpEventResponse handle(HttpEventRequest request) {
      *
      * @param request
      */
-    void dynamicallyConfigureSanitizer(HttpEventRequest request) {
+    Sanitizer getSanitizerForRequest(HttpEventRequest request) {
         Optional<PseudonymImplementation> pseudonymImplementation = parsePseudonymImplementation(request);
         if (pseudonymImplementation.isPresent()) {
+            loadSanitizerRules(); // ensure sanitizer is loaded
```

Comment:
you can remove this, `loadSanitizerRules` is one of the very first calls in the handler, so no point in doing it again.

Reply:
tests req'd it